runtime/crab/src/pallets/staking.rs

```diff
@@ -70,8 +72,25 @@ impl darwinia_staking::Stake for KtonStaking {
 	}
 }
 
-frame_support::parameter_types! {
-	pub const PayoutFraction: sp_runtime::Perbill = sp_runtime::Perbill::from_percent(40);
+pub enum OnCrabSessionEnd {}
+impl darwinia_staking::OnSessionEnd<Runtime> for OnCrabSessionEnd {
+	fn calculate_reward(_maybe_inflation: Option<Balance>) -> Balance {
+		20_000 * UNIT
+	}
+
+	fn reward(who: &AccountId, amount: Balance) -> sp_runtime::DispatchResult {
+		if let Err(e) = <Balances as Currency<AccountId>>::transfer(
+			&Treasury::account_id(),
+			who,
+			amount,
+			frame_support::traits::ExistenceRequirement::KeepAlive,
+		) {
+			// TODO: log
+			// log::error!("[runtime::staking] reward error: {:?}", e);
+		}
+
+		Ok(())
```

Comment:
Currently, the reward here always returns `Ok()`, it seems problematic. If the transfer fails, the `n_payout` should not included in the `actual_payout`.